<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoMetadataDialogBase</class>
 <widget class="QDialog" name="GeoMetadataDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>641</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GeoMetadata</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Título:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_date_creation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data de Criação</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_date_creation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>12</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1800</year>
     <month>9</month>
     <day>14</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_edition">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Edição</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_abstract">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Resumo</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_abstract">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Resumo descritivo da camada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_codeListValue">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_status_codeListValue">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_contact_presets">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o setor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DADOS DE CONTATO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MD_Keywords_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Palavra-chave</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MD_Keywords">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Palavra-chave 1, Palavra-chave 2, Palavra-chave 3...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MD_SpatialRepresentationTypeCode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o tipo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MD_SpatialRepresentationTypeCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de Representação Espacial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spatialResolution_denominator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Escala</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_LanguageCode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o idioma</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LanguageCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Idioma</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_characterSet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o Encoding</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_characterSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Codificação</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_topicCategory">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione a categoria</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_topicCategory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria Temática</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_westBoundLongitude">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-53.11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_westBoundLongitude">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Oeste</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_northBoundLatitude">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Norte</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_southBoundLatitude">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sul</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_eastBoundLongitude">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Leste</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_northBoundLatitude">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-19.7667</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_southBoundLatitude">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-25.3333</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eastBoundLongitude">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-44.1667</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_hierarchyLevel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nível Hierárquico</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_hierarchyLevel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione o tipo da camada</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_dateStamp">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>600</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>12</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1800</year>
     <month>9</month>
     <day>14</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_dateStamp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data do metadado</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_edition">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exp_xml">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar para XML</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exp_geo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar para o Geohab</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_textEdit_spatialResolution_denominator">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>5000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Título da camada</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_preencherbbox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preencher BoundBox</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_individualName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>CDHU</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_organisationName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Companhia de Desenvolvimento Habitacional e Urbano</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_positionName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_phone">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>+55 11 0000-0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_postalCode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>01014-930</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_country">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Brasil</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_email">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>cdhu@cdhu.sp.gov.br</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_city">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>São Paulo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contact_deliveryPoint">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Rua Boa Vista, 170, Sé</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome/Sigla</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome completo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Posição</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rua, nº - complemento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CEP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>País</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_label_label_label_contact_individualName_11">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Responsabilidade</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_contact_role">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_contact_administrativeArea">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Selecione</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
